### ISSUE TITLE
endworm gets stuck on the map's ceiling

### ISSUE THREAD

**Issue by LePiromano** [Regalis11/Barotrauma#8468]
- [yes] I have searched the issue tracker to check if the issue has already been reported.

**Description**
![image](https://user-images.githubusercontent.com/55637865/156945438-922e8e18-6568-4499-b7e0-011ba4567aae.png)
(the endworm won't move)

**Version**
v0.17.0

**Comment by LePiromano** [comment]
It's not only the map ceiling the endworm can get stuck on, since it's head has a flat hitbox and cant turn very fast,  but It getting stuck on the ceiling is very common (at least for me)

**Comment by itchyOwl** [comment]
Is it the ceiling of the abyss or the ceiling of the entire level? If the latter, how did it get there?

**Comment by LePiromano** [comment]
> Is it the ceiling of the abyss or the ceiling of the entire level? If the latter, how did it get there?

The later, you go to the right or left side of the level then go up after triggering It and It will follow

**Comment by itchyOwl** [comment]
I don't think we should remove the issue. It's very low prio, but still a clear bug. The monsters should be able to avoid the level walls, but I guess the raycast isn't working in the level ceiling or something. Moving back to bugfixes.
